conda_smithy/lint_recipe.py

```diff
@@ -86,7 +88,13 @@ def lintify(meta, recipe_dir=None):
     if 'unknown' == license.strip():
         lints.append('The recipe license cannot be unknown.')
 
-    # 6: Selectors should be in a tidy form.
+    # 6: License family must be valid (conda-build checks for that)
```

Comment:
I don't know that we want to be messing with the number. Maybe @pelson can comment.